pom.xml

```diff
@@ -125,7 +125,7 @@
         <gravitee-policy-ipfiltering.version>1.8.0</gravitee-policy-ipfiltering.version>
         <gravitee-policy-request-validation.version>1.12.0</gravitee-policy-request-validation.version>
         <gravitee-policy-latency.version>1.4.0</gravitee-policy-latency.version>
-        <gravitee-cockpit-connectors.version>4.0.2-alpha.1</gravitee-cockpit-connectors.version>
+        <gravitee-cockpit-connectors.version>4.0.2</gravitee-cockpit-connectors.version>
```

Comment:
Cockpit connector 4.0.2 seems to be linked to gravitee node 4.0.0 & gravitee plugin 2.0.1 
https://github.com/gravitee-io/gravitee-cockpit-connectors/blob/master/pom.xml#L43

Does cockpit connection has been tested with this plugin ?

Reply:
Tested locally by the team :heavy_check_mark: